### ISSUE TITLE
Setting `openai.api_key` panics

### ISSUE THREAD

**Issue by andreyfedoseev** [zurawiki/gptcommit#33]
I have a fresh install of `gptcommit-0.1.9`

When I'm trying to set `openai.api_key` like so:

```
gptcommit config set openai.api_key sk-…
```

it fails with this error:

```
thread 'main' panicked at 'Could not find user config path', /home/…/.cargo/registry/src/github.com-1ecc6299db9ec823/gptcommit-0.1.9/src/actions/config.rs:85:51
```

**Comment by zurawiki** [comment]
Acknowledged the issue. I'll push a fix in 0.1.10.

Thanks for reporting!

**Comment by vitalwarley** [comment]
It works properly now. Thanks!
